### ISSUE TITLE
Pagination support

### ISSUE THREAD

**Issue by jaydson** [JSRocksHQ/harmonic#5]
Harmonic must have pagination support for posts.

**Comment by jaydson** [comment]
Hum, this issue is old, but remains important.

**Comment by UltCombo** [comment]
True, we haven't decided how to generate the output and how to integrate it in the templates.
